index.html

```diff
@@ -344,7 +342,6 @@ <h1>Project Name</h1>
             </div>
         </div>
     
-    
     <script src="./index.js"></script>
```

Comment:
- [ ] According to the project requirement, The contact form should contain at least one validation checking if the email is written in all lowercase: 🤔 
   * If a user submits the contact form with the email field containing uppercase characters an error message should show up close to the "submit" button.  kindly make this 😉 
ie: `email === email.toLowerCase()`
- Kindly make sure that The JavaScript code is implemented following the [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).The JavaScript code is implemented following the [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).